packages/workshop-app/app/components/diff.tsx

```diff
@@ -171,11 +171,11 @@ export function Diff() {
 						</div>
 						<div className="scrollbar-thin scrollbar-thumb-gray-300 max-h-[calc(100vh-109px)] overflow-y-auto">
 							{diff.diffCode ? (
-								<div>
+								<h2>
```

Comment:
this is actually a tip from Lighthouse, it is because the `Accordion` uses h3.
```ts
  <h2>
	  <Accordion.Root className="w-full" type="multiple">
		  <Mdx code={diff.diffCode} components={mdxComponents} />
	  </Accordion.Root>
  </h2>
```

Reply:
I've never heard of wrapping a bunch of content in a heading. The solution is definitely not to wrap it in an `h2` but instead to either change what `Accordion` renders or render an `h2` elsewhere on the page.

Reply:
my bad, I've missed it in the [docs](https://www.radix-ui.com/docs/primitives/components/accordion#header)